rasa/cli/train.py

```diff
@@ -16,7 +18,7 @@
     DEFAULT_DOMAIN_PATH,
     DEFAULT_DATA_PATH,
 )
-import rasa.utils.common
+from rasa.train import train as train_all, train_core, train_nlu
```

Comment:
this will probably be fixed by Tom [here](https://github.com/RasaHQ/rasa/pull/8141) but can we please use local imports here to avoid slow imports?